server/monitor/src/main/java/org/apache/accumulo/monitor/rest/trace/TracesResource.java

```diff
@@ -280,10 +279,6 @@ protected Range getRangeForTrace(long minutesSince) {
     long startTime = endTime - millisSince;
 
     String startHexTime = Long.toHexString(startTime), endHexTime = Long.toHexString(endTime);
-    if (startHexTime.length() < endHexTime.length()) {
-      StringUtils.leftPad(startHexTime, endHexTime.length(), '0');
```

Comment:
Was this just a no-op? Was this supposed to assign to startHexTime or endHexTime after padding?

Reply:
Yeah it wasn't doing anything.  Not sure what it was "supposed" to be doing...